### PR TITLE
Local run get

### PR DESCRIPTION
**Pull request by melsomino**

### CODE REVIEW

File: ton_sdk/src/contract.rs [hunk 1]
```diff
@@ -51,6 +52,110 @@ use ton_block::TransactionProcessingStatus;
...
+                } else if s.starts_with("-0x") || s.starts_with("-0X") {
+                    Some(1)
+                } else {
+                    None
+                };
+            if let Some(pos) = hex_prefix_pos {
+                let mut ms = s.clone();
+                ms.remove(pos);
+                ms.remove(pos);
+                IntegerData::from_str(ms.as_str())?
```
Comment (AlexeyVavilin) [inline ton_sdk/src/contract.rs:124]: `IntegerData::from_str(&s[pos..])?` and remove `ms` variable

File: ton_client/client/src/tests/mod.rs [hunk 2]
```diff
@@ -106,6 +108,117 @@ fn get_map_string(m: &Map<String, Value>, f: &str) -> String {
...
+    let _ = log4rs::init_file(log_cfg_path, Default::default());
+
+    let client = TestClient::new();
+    let result = parse_object(client.request("tvm.get", json!({
+        "functionName": "participant_list",
+        "codeBase64": ELECTOR_CODE,
+        "dataBase64": ELECTOR_DATA,
+    })));
+
+    println!("{}", serde_json::Value::Object(result).to_string())
```
Comment (AlexeyVavilin) [inline ton_client/client/src/tests/mod.rs:125]: Need to compare with ethalon value

File: ton_client/client/src/client.rs [hunk 3]
```diff
@@ -27,7 +27,9 @@ fn create_handlers() -> DispatchTable {
     crate::setup::register(&mut handlers);
     crate::crypto::register(&mut handlers);
     crate::contracts::register(&mut handlers);
-    
+    crate::tvm::register(&mut handlers);
+    crate::cell::register(&mut handlers);
```
Comment (AlexeyVavilin) [inline ton_client/client/src/client.rs:31]: Remove registration as theese functions are not completed yet

File: ton_sdk/src/contract.rs [hunk 4]
```diff
@@ -610,10 +715,47 @@ impl Contract {
         Ok(messages)
     }
 
+
+    /// Invokes local TVM instance with provided stack.
+    /// Returns stack after contract execution.
+    /// Used for get methods
+    pub fn local_call_tvm_get_json(
+        &self,
+        function_name: &String,
```
Comment (AlexeyVavilin) [inline ton_sdk/src/contract.rs:724]: Borrowed strings are usually passed as `&str`. Check all occurencies

File: ton_sdk/src/contract.rs (hunk 1 above)
Comment (melsomino) [reply ton_sdk/src/contract.rs:124]: It's not the same.
"-0xabcd" must convert to "-abcd"

**Pull request merged**